engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -29,14 +28,17 @@
  * TODO: Add javadocs
  */
 public final class SetWireframe implements StateChange {
+    private static final int ENABLED_MODE = GL_LINE;
+    private static final int DISABLED_MODE = GL_FILL;
```

Comment:
I am not convinced about these two variables - they made more sense in the task perhaps. Here I'd recommend to use the opengl constants directly.